src/Uno.UWP/UI/Input/GestureRecognizer.Manipulation.cs

```diff
@@ -705,7 +705,7 @@ private void UpdateComputedValues()
 						var p2 = _pointer2.RawPosition;
 
 						Center = new Point((p1.X + p2.X) / 2, (p1.Y + p2.Y) / 2);
-						Distance = Vector2.Distance(p1.ToVector(), p2.ToVector());
```

Comment:
Do we have pointer tests to account for this change?

Reply:
Yes we do have (Unit) tests for the `GestureRecognizer`. This is however just a rename of the method to remove duplicate.